### ISSUE TITLE
Replacing "get in touch" mailto links with link to our sign-up form

### ISSUE THREAD

**Issue by 1l2p** [ml-learning/plix-staging#22]
Is there an easy way to send people to the sign-up form we have been using before the launch of the site? I'm not sure if the form content need to be updated, but it's nice to have a spreadsheet with everyone who has expressed an interest, rather than individual emails.

**Comment by hakbailey** [comment]
Yeah, I agree that a form would be easier to track. Some of the places we have email links are for more specific purposes though, like asking questions and giving feedback. What if I make a generic "contact us" form to use on those pages instead of the interest form?

I think it would be great to still include the interest form too, maybe on the "community" page? And phrase the link to make it clear this form is different from the contact form.

Another option would be to use one form, but have the first page be a multiple choice question to figure out why they're contacting us, and then conditional logic to show different follow up questions based on that.

**Comment by hakbailey** [comment]
Ok, I made a new contact form and replaced the mailto links with it. Leaving this issue open until we decide whether/where/how to also include the interest form, which gathers more info.

**Comment by kitsandkats** [comment]
@hakbailey -- do the links go to a new sign-up form now? Or the same one that we made before? 

I am getting submissions to our old survey, as well as a few emails that say "Hey PLIX Team, someone wants to contact you!" -- I assume that these are different, but I am not sure where each is coming from...

**Comment by hakbailey** [comment]
The old form is linked from our latest blog post, which is why we're getting so many responses to it (yay!). The new form is used in all the contact links on the website, and is what's generating those "someone wants to contact you" messages. I made the email notifications different partially to make it easy for us to distinguish them.

**Comment by 1l2p** [comment]
Would it be possible to just pipe them all into one form? (and update the
form if necessary)

It's nice to have everything in one place... P
